### PR TITLE
First iteration of a simple ManagedIdentity integration test

### PR DESCRIPTION
**Pull request by christothes**
related #37471

### CODE REVIEW

File: sdk/identity/test-resources-post.ps1 [hunk 1]
```diff
@@ -0,0 +1,19 @@
+param (
+    [hashtable] $DeploymentOutputs
+)
+
+$webappRoot = "$PSScriptRoot/../../sdk/identity/Azure.Identity/integration" | Resolve-Path
```
Comment (benbp) [inline sdk/identity/test-resources-post.ps1:5]:
```suggestion
$webappRoot = "$PSScriptRoot/Azure.Identity/integration" | Resolve-Path
```

File: sdk/identity/tests.yml [hunk 2]
```diff
@@ -1,36 +1,12 @@
 trigger: none
 
-resources:
```
Comment (benbp) [inline sdk/identity/tests.yml:3]: Can we delete the dockerfile from `eng/containers` as well?

**Review by benbp — APPROVED** [review]
eng/deploy script changes look good.

File: sdk/identity/tests.yml (hunk 2 above)
Comment (christothes) [reply sdk/identity/tests.yml:3]: I think we may need it in the future, so do you mind if we leave it around for now?

File: sdk/identity/Azure.Identity/tests/VisualStudioCodeCredentialTests.cs [hunk 3]
```diff
@@ -12,14 +12,15 @@
...
+    [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/27263")]
     public class VisualStudioCodeCredentialTests : CredentialTestBase
     {
         public VisualStudioCodeCredentialTests(bool isAsync) : base(isAsync)
         { }
 
         public override TokenCredential GetTokenCredential(TokenCredentialOptions options)
         {
-            using var env = new TestEnvVar(new Dictionary<string, string> { { "TENANT_ID", TenantId } });
+            using var env = new TestEnvVar(new Dictionary<string, string> { { "IDENTITY_TENANT_ID", TenantId }, { "TENANT_ID", TenantId } });
```
Comment (schaabs) [inline sdk/identity/Azure.Identity/tests/VisualStudioCodeCredentialTests.cs:23]: nit: do we still need to set `TENANT_ID`

**Review by schaabs — APPROVED** [review]

**Pull request merged**